meta-xt-driver-domain/recipes-extended/sndbe/sndbe_git.bb

```diff
@@ -5,8 +5,8 @@ PR = "r0"
 
 inherit pkgconfig cmake systemd
 
-DEPENDS = "libxenbe libconfig pulseaudio git-native"
-RDEPENDS_${PN} = "libxenbe libconfig pulseaudio-server"
```

Comment:
You need to add dependency for pulseaudio in case if `DISTRO_FEATURES` enables it.

Reply:
Fixed. @lorc p.lease review.

Reply:
Well, this does not look good. I believe you need to use PACKECONFIG as other recipes do.

This will cover `EXTRA_OECMAKE`, `DEPENDS` and `RDEPENDS` in one go. For example, take a look there:
https://git.yoctoproject.org/poky/tree/meta/recipes-gnome/gtk+/gtk+3.inc#n55

Reply:
Agree. That's better approach. Refactored code.